app/assets/stylesheets/utilities/colours.scss

```diff
@@ -1,6 +1,6 @@
-$colour-black: govuk-colour("black");
```

Comment:
Can you please add a card to tech debt for refactoring and add an item to see if we can get rid of this file?

Reply:
Sure.